modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -88,7 +88,7 @@ Learn how to calculate resources, such as memory and CPU, required to run {prod}
 >s|6.4 GiB
 >s|640 MiB
 >s|4550 m
->s|1060 m
+>s|2050 m
```

Comment:
This `CPU limit` total is fine as it is. It's the total on the previous line (`CPU limit`) that needs to be updated to `5500 m`.